src/pwm.rs

```diff
@@ -28,227 +28,157 @@
 
     // Put the timer in PWM mode using the specified pins
     // with a frequency of 100 hz.
+    // In some cases you need to specify remap you need, especcially for TIM2:
```

Comment:
Would be nice to point users to where they can find what remaps they can specify.

Also double 'c' in especially